llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -498,13 +498,15 @@ void VPWidenCallRecipe::execute(VPTransformState &State) {
          "DbgInfoIntrinsic should have been dropped during VPlan construction");
   State.setDebugLocFrom(CI.getDebugLoc());
 
+  Intrinsic::ID ID = VectorIntrinsicID;
```

Comment:
Is it worth to create a local variable ? IMO it is better to use the "VectorIntrinsicID"

Reply:
My argument is that it helps with code indentation.

Reply:
Done